multimedia/views.py

```diff
@@ -4,6 +4,8 @@
 )
 from crew.models import Crew
 
+no_preview_150 = '/statics/images/no-preview-150.png'
+no_preview = '/statics/images/no-preview.png'
```

Comment:
:+1: as long as we don't hard code the URL.
For Static reverse we can use the templatetags at staticfiles.
`django.contrib.staticfiles.templatetags.staticfiles.static`

    from django.contrib.staticfiles.templatetags.staticfiles import static
    no_preview = static('images/no-preview.png')